src/Jobs/ReadChunk.php

```diff
@@ -97,6 +98,10 @@ public function __construct(WithChunkReading $import, IReader $reader, Temporary
      */
     public function handle(TransactionHandler $transaction)
     {
+        if ($this->import instanceof WithChunkOffset) {
```

Comment:
Perhaps to simplify this, we could just check of method existence, so no concern needs to be added for this?